libs/frontend/feature-build/src/components/build-render/ProseMirrorEditor.vue

```diff
@@ -70,7 +71,10 @@ const gradientColorStyle = computed(() => {
     activeBreakpoint.value,
   )
 
-  const background = gradientColorStyles[Css.Background]
+  const background = resolveThemeVariables(
```

Comment:
Now the prosemirror style has to resolve theme variables in the background color.